### PR TITLE
docs: add spatial system and typography stories

### PR DESCRIPTION
**Pull request by ricardogoncalves89**
image
image

### CODE REVIEW

**Review by miguelpinto98 — APPROVED** [review]
👍

**Review by djcouto — APPROVED** [review]
🚀

File: packages/react-ui-components/src/stories/foundation/Typography/Typography.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+const paragraphs = [
+	'kv-font-paragraph-regular',
+	'kv-font-paragraph-light'
+];
+
+< Meta title='Foundation/Typography'/>
+
+# Typography
+
+*Kelvin inc* primary typeface is Proxima Nova. **Proxima Nova** was designed to combine modern proportions with a geometric appearance. It allows to create and build interfaces with a look and feels with a touch of user-friendly, contemporary, fresh, and timeless.
```
Comment (pedrorebelocosta) [inline packages/react-ui-components/src/stories/foundation/Typography/Typography.stories.mdx:39]: *Kelvin's* primary typeface is Proxima Nova. **Proxima Nova** was designed to combine modern proportions with a geometric appearance. It allows you to build interfaces with a look and feel that combines a touch of user-friendly, contemporary, fresh, and timeless design.

File: packages/react-ui-components/src/stories/foundation/Typography/Typography.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+];
+
+< Meta title='Foundation/Typography'/>
+
+# Typography
+
+*Kelvin inc* primary typeface is Proxima Nova. **Proxima Nova** was designed to combine modern proportions with a geometric appearance. It allows to create and build interfaces with a look and feels with a touch of user-friendly, contemporary, fresh, and timeless.  
+We will use the typeface at various weights, primarily *Light [300]*, *Regular [400]*, *Semibold [600]*, and *Bold [700]*.  
+
+**Note**: Each variation has yours `@mixin` available for use. For that you need to import the `kv-typography.scss` file.
```
Comment (pedrorebelocosta) [inline packages/react-ui-components/src/stories/foundation/Typography/Typography.stories.mdx:42]: **Note**: Each variation has a `@mixin` available for use. In order to use those _mixins_, you'll need to import the `kv-typography.scss` file.

**Review by pedrorebelocosta — APPROVED** [review]

**Review by pedrorebelocosta — CHANGES_REQUESTED** [review]
Please check the comments on the mdx files.

**Review by pedrorebelocosta — APPROVED** [review]

**Pull request merged**